IFC4x3/Properties/n/NumberOfPoles_2lqxx3IET2UhUPt$MJGW_4/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NumberOfPoles_2lqxx3IET2UhUPtMJGW_4" Name="NumberOfPoles" UniqueId="afd3bec3-48e7-427a-b799-dff593420f84" PrimaryDataType="IfcCountMeasure" />
+
```

Comment:
Should this really be IfcCountMeasure? Just checking (i had some comments about this before, but I think that it was decided to use IfcInteger).

Reply:
Please look into this comment. Can I resolve?

Reply:
Changed to count Measure. Commit : b5e658cad21e4944423ed95d945268c722299c68

Reply:
Are you sure that it shouldn't be IfcInteger?